.changes/unreleased/1.3.0.md

```diff
@@ -0,0 +1,80 @@
+## dbt-snowflake 1.3.0 (TBD)
+
+### Under the hood
+- Reformat overridden macro location of grants work to a apply_grants.sql file in snowflake ([#193](https://github.com/dbt-labs/dbt-snowflake/issues/193), [#192](https://github.com/dbt-labs/dbt-snowflake/pull/192))
+- Support dbt Core incremental materialization refactor ([#195](https://github.com/dbt-labs/dbt-snowflake/issues/195), [#196](https://github.com/dbt-labs/dbt-snowflake/pull/196))
```

Comment:
I would actually convert these to be changie yaml files in `unreleased` since `1.3.0-b1` is not actually released yet.  Then these can persist as we do rcs and the final 1.3 release.

Then you can delete this file completely and let it get generated with `changie batch...` when we release the beta.

Reply:
converted all but one entry over trying to make sure the remaining one is one we want to swap over.